warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/IngestJob.java

```diff
@@ -682,7 +682,7 @@ protected Configuration parseArguments(String[] args, Configuration conf) throws
             } else if (args[i].equals("-cacheBaseDir")) {
                 cacheBaseDir = args[++i];
             } else if (args[i].equals("-cacheJars")) {
-                String[] jars = StringUtils.trimAndRemoveEmptyStrings(args[++i].split("\\s*,\\s*"));
+                String[] jars = StringUtils.trimAndRemoveEmptyStrings(args[++i].replaceAll("\\s+", "").split("\\s{0,1},\\s{0,1}"));
```

Comment:
Doesn't `.replaceAll("\\s+", "")` make `split(",")` sufficient?

Reply:
it does -- fixed.